### ISSUE TITLE
Add a new FAQ session to README

### ISSUE THREAD

**Issue by zhenying-liu** [google/jax#17296]
When I run my jax code, I got a "bus error". It was caused by the size of /dev/shm too small. Adding --shm-size=1g to the container launch command line works. For example,
       docker run -it --shm-size=1g ...
Suggest documenting this kind of issues in FAQ.

**Comment by SainiAditya1** [comment]
I want to work on assign me this .

**Comment by jakevdp** [comment]
I've not seen this issue come up before, so it may not rise to the level of a "frequently" asked question.

I'd suggest pasting the full error message here, so when people Google it they'll find the solution.

**Comment by hawkinsp** [comment]
I have seen this issue come up before when running JAX on CUDA inside a docker container. It might merit a FAQ entry.

**Comment by zhenying-liu** [comment]
Below is the complete error message when I tried to work on cudaAsynMemoryAlloc with a code example from flax/examples/imagenet/imagenet_fake_data_benchmark.py.
```
Fatal Python error: Fatal Python error: Bus errorBus error
Thread 0x00007f50e814d000 (most recent call first):
  File "/opt/jax-source/jax/_src/interpreters/pxla.py", line 1158 in __call__
  File "/opt/jax-source/jax/_src/profiler.py", line 314 in wrapper
  File "/opt/jax-source/jax/_src/api.py", line 1803 in cache_miss
  File "/opt/jax-source/jax/_src/traceback_util.py", line 177 in reraise_with_filtered_traceback
  File "/workspace/flax/examples/imagenet/train.py", line 380 in train_and_evaluate
  File "/workspace/flax/examples/imagenet/imagenet_fake_data_benchmark.py", line 50 in test_fake_data
```
  Also we want to add the FAQ to jax-tool as well. I made a pull request there for issue #187 of adding FAQ to readme. https://github.com/NVIDIA/JAX-Toolbox/pull/187.

**Comment by ayaka14732** [comment]
I don't think this should be added to the JAX FAQ. This is definitely not a _frequently_ asked question, and it is a issue about environment setup, which is not a JAX issue at all. I believe the correct way to solve the issue is to modify your Docker image configuration.
